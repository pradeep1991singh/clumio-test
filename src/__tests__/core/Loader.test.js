import React from "react";
import { render } from "@testing-library/react";
import Loader from "../../core/Loader";

describe("<Loader />", () => {
  test("Snapshot", () => {
    const { asFragment } = render(<Loader />);
    expect(asFragment()).toMatchSnapshot();
  });
});
