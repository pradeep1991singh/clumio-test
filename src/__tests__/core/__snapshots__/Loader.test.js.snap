// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> Snapshot 1`] = `
<DocumentFragment>
  <button
    class="bg-indigo-500 ..."
    disabled=""
    type="button"
  >
    <svg
      class="animate-spin h-5 w-5 mr-3 ..."
      viewBox="0 0 24 24"
    />
    Loading...
  </button>
</DocumentFragment>
`;
